virtualenv.py

```diff
@@ -1620,7 +1621,17 @@ def install_python(home_dir, lib_dir, inc_dir, bin_dir, site_packages, clear, sy
         if "EPD" in prefix:
             logger.debug("EPD framework detected")
             original_python = os.path.join(prefix, "bin/python")
-        shutil.copy(original_python, py_executable)
+        original_maxsize = int(
```

Comment:
this part to feels a very heavyweight; the way things work feels to me that the copy is that original_python is bad during reinvocation at the start; so that should be fixed not this

Reply:
This is more of an issue. I understand that the current master branch of `virtualenv` feels bad at the start. But unfortunately, I don't have time to commit on working on this task which is much more that a small fix and needs understanding why this was done in the first place in order to ensure not breaking anything.